docs/tutorials/java/build-an-email-drip-campaign/index.md

```diff
@@ -1,22 +1,22 @@
 ---
 id: subscription-tutorial
 sidebar_position: 3
-keywords: [Java, temporal, sdk, tutorial, entity workflow, email drip campaign, email subscription, sending emails]
+keywords: [Java, Spring Boot, temporal, sdk, tutorial, entity workflow, email drip campaign, email subscription, sending emails]
 tags: [Java, SDK]
 last_update:
   date: 2024-06-27
-title: Build an email drip campaign with Temporal and Java
+title: Build an email drip campaign with Spring Boot and Java
 description: Implement an email drip campaign application with Temporal's Workflows, Activities, and Queries, and allow users to start your business logic through a web action.
-sidebar_label: Build an email drip campaign with Temporal and Java
+sidebar_label: Build an email drip campaign with Spring Boot and Java
```

Comment:
if they are the same, we can delete.

Reply:
You said they should always align. They are aligned. We can delete.